Cohere/Types/ChatRequest.cs

```diff
@@ -0,0 +1,90 @@
+namespace Cohere.Types;
+
+/// <summary>
+/// The object to send to the chat endpoint
+/// </summary>
+public class ChatRequest: ICohereRequest
```

Comment:
you can separate the `Types` folder into multiple folders or sub-folders. 
Request, Response, Enum, Model,